pennylane/templates/embeddings.py

```diff
@@ -136,15 +140,22 @@ def AmplitudeEmbedding(features, wires):
     Args:
         features (array): Input array of shape ``(2**n,)``
         wires (Sequence[int]): sequence of qubit indices that the template acts on
+        pad (Boolean): controls the activation of the padding option, defaults to ``False``
     """
 
     if not isinstance(wires, Iterable):
         raise ValueError("Wires needs to be a list of wires that the embedding uses; got {}.".format(wires))
 
-    if 2**len(wires) != len(features):
-        raise ValueError("AmplitudeEmbedding requires a feature vector of size 2**len(wires), which is {}; "
+    if pad and 2**len(wires) != len(features):
+        features = np.pad(features, (0, 2**len(wires)-len(features)), 'constant')
```

Comment:
I think this will raise an uninformative error if the user gives more features than the wires can handle (`len(features) > 2**len(wires)`) and `pad = True.`

You should probably add another if that checks if `len(features) > 2**len(wires)`

Reply:
ah good point!